ffcx/codegeneration/C/integrals.py

```diff
@@ -56,6 +56,14 @@ def generator(ir, options):
     code["additional_includes_set"] = set()  # FIXME: Get this out of code[]
     code["tabulate_tensor"] = body
 
+    code["tabulate_tensor_float32"] = "NULL"
+    code["tabulate_tensor_float64"] = "NULL"
+    code["tabulate_tensor_longdouble"] = "NULL"
```

Comment:
We should remove all the `long double' and stick with properly defined types. Ill-defined, platform-dependent types cause nothing but trouble.